src/v3/src/transformer/testAttribute/transform.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { flow } from 'lodash';
+
+import {
+  ButtonElement,
+  ButtonType,
+  FieldElement,
+  LinkElement,
+  TransformStepFn,
+} from '../../types';
+import { updateElementsInLayout } from '../util';
+
+const updateFields: TransformStepFn = (formbag) => {
+  const { uischema } = formbag;
+
+  updateElementsInLayout({
+    layout: uischema,
+    mapFn: (element) => {
+      const { options } = element as FieldElement;
+      options.dataSe = options.inputMeta.name;
+      return element;
+    },
+    predicateFn: (element) => {
+      const { type } = element as FieldElement;
+      return type === 'Field';
+    },
+  });
+
+  return formbag;
+};
+
+const updateSubmitButton: TransformStepFn = (formbag) => {
+  const { uischema } = formbag;
+
+  updateElementsInLayout({
+    layout: uischema,
+    mapFn: (element) => {
+      const { options } = element as ButtonElement;
+      options.dataType = 'save';
+      return element;
+    },
+    predicateFn: (element) => {
+      const { type, options } = element as ButtonElement;
+      return type === 'Button' && options.type === ButtonType.SUBMIT;
+    },
+  });
+
+  return formbag;
+};
+
+const updateRegularButtons: TransformStepFn = (formbag) => {
+  const { uischema } = formbag;
+
+  const map: Record<string, string> = {
+    'select-enroll-profile': 'enroll',
+    'currentAuthenticator-recover': 'forgot-password',
+    'currentAuthenticatorEnrollment-recover': 'forgot-password',
+    'unlock-account': 'unlock',
+    cancel: 'cancel',
+    'select-identify': 'back',
+    'select-authenticator-authenticate': 'switchAuthenticator',
+    'select-authenticator-enroll': 'switchAuthenticator',
+  };
+
+  updateElementsInLayout({
+    layout: uischema,
+    mapFn: (element) => {
+      const { options } = element as ButtonElement;
+      options.dataSe = map[options.step];
+      return element;
+    },
+    predicateFn: (element) => {
+      const { type, options } = element as ButtonElement | LinkElement;
+      return (type === 'Button' && (options as ButtonElement['options']).type === ButtonType.BUTTON)
+        || type === 'Link';
+    },
+  });
+
+  return formbag;
+};
+
+export const transformTestAttribute: TransformStepFn = (formbag) => flow(
```

Comment:
Are there any unit tests that cover this transform?

Reply:
Planning to add tests in later stage, currently transformer tests are disabled in jest config to avoid noise from existing tests.
I can make some config change if you have strong feeling to include tests in this PR.

Reply:
I actually fixed all the transformer tests in the last push to 0.0, so you could really just re-enable them and they should not fail (outside of the few cases that were updated in this branch).

Reply:
those custom transformer will be changed in later stage, don't want to bother with touching all those files for now